configs/openmrs_config/concepts/past_medical_history.csv

```diff
@@ -12,11 +12,11 @@ fe0b5b8e-c9a2-4c87-813d-f10884987d9c,,,Previous diabetes mellitus diagnosis?,ក
 3fe4ac96-893f-4193-a896-1b7e9528719d,,,Year of diagnosis,ឆ្នាំនៃការធ្វើរោគវិនិច្ឆ័យ,Year of diagnosis,ឆ្នាំនៃការធ្វើរោគវិនិច្ឆ័យ,,,Misc,Date,,,,
 d3c8a8d4-c141-401f-8354-adbeda86011f,,,High Blood Pressure Symptoms,រោគសញ្ញាលើសសម្ពាធឈាម,High Blood Pressure Symptoms,រោគសញ្ញាលើសសម្ពាធឈាម,,,Misc,Text,,,,
 4aea53d6-88d1-4e4c-8702-a8142366af02,,,Diet Only,របបអាហារតែមួយគត់,Diet Only,របបអាហារតែមួយគត់,,,Misc,N/A,,,,
-8c0fc1cc-f430-4201-968b-db0071b01dfd,,,Oral Hypoglycaemics,ឧិសថលេបបញ្ចុះជាតិស្ករ,Oral Hypoglycaemics,ឧិសថលេបបញ្ចុះជាតិស្ករ,,,Misc,N/A,,,,
-d99b139e-4d8f-462c-9825-a0cdc0824417,,,Insulin therapy,ការព្យាបាលដោយអាំងស៊ុយលីន,Insulin therapy,ការព្យាបាលដោយអាំងស៊ុយលីន,,,Misc,N/A,,,,
-c4c14717-6624-4963-b038-863eb51669a8,,,Oral Hypoglycaemics and Insulin therapy,ការព្យាបាលដោយឧិសថលេបបញ្ចុះជាតិស្ករ និងការព្យាបាលដោយអាំងស៊ុយលីន,Oral Hypoglycaemics and Insulin therapy,ការព្យាបាលដោយឧិសថលេបបញ្ចុះជាតិស្ករ និងការព្យាបាលដោយអាំងស៊ុយលីន,,,Misc,N/A,,,,
+8c0fc1cc-f430-4201-968b-db0071b01dfd,,,Oral Hypoglycaemics Agent,,Oral Hypoglycaemics Agent,,,,Misc,N/A,,,,
+d99b139e-4d8f-462c-9825-a0cdc0824417,,,"Insulin ",អាំងស៊ុលលីន,Insulin,អាំងស៊ុលលីន,,,Misc,N/A,,,,
```

Comment:
Remove the space and thus the quotes.

Reply:
@rbuisson , This has been worked on and the Khmer translation added